esrally/tracker/tracker.py

```diff
@@ -76,6 +88,14 @@ def create_track(cfg):
     output_path = os.path.abspath(os.path.join(io.normalize_path(root_path), track_name))
     io.ensure_dir(output_path)
 
+    if datastreams is None:
+        logger.info("Creating track [%s] matching indices [%s]", track_name, indices)
+    else:
+        logger.info("Creating track [%s] matching datastreams [%s]", track_name, datastreams)
+        extracted_indices = extract_indexes_from_datastreams(client, datastreams)
+        indices = extracted_indices
+        console.info(f"##### List datastreams to extract [{indices}]", logger=logger)
```

Comment:
So are you listing datastreams or indices?

Reply:
Updated with indexes as above 
`logger.info("Creating track [%s] matching indices [%s]", track_name, indices)`